src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useState } from 'react';
```

Comment:
You should be consistent in your imports too. You shouldn't need to import twice from the same package.

Reply:
Done!